fix(ui links): expose full path to UI (#13987)
newrelic/docs-website: src/content/docs/apm/apm-ui-pages/monitoring/view-slow-query-details.mdx

To see your slow query data:

1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **Databases**.

> 2. Select a database transaction.[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
> 3. If available, s[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >the page.

Author: ```suggestion
2. Select a database transaction.
3. If available, select any available [slow queries](#slowsql_details) listed on the page.
```
